### DISCUSSION TITLE
quarto publish missing on Ubuntu LTS 22.04

### DISCUSSION THREAD

**Discussion by eyayaw** [quarto-dev/quarto-cli#1638] [Q&A] [answered]
I have been trying to publish to GitHub Pages using the quarto cli as explained [here](https://quarto.org/docs/publishing/github-pages.html). However, my shells seems not to find the `quarto publish` command:

```fish
quarto publish gh-pages
ERROR: Unknown command "publish". Did you mean command "preview"?
```

![image](https://user-images.githubusercontent.com/49317723/182337351-a4dbf280-1e20-4979-b6fc-f1a0bfff08b5.png)

Is the command yet unavailable, or am I missing something? 

```fish
quarto check
```

````
[✓] Checking Quarto installation......OK
      Version: 0.9.419
      Path: /opt/quarto/bin

[✓] Checking basic markdown render....OK

[✓] Checking Python 3 installation....OK
      Version: 3.10.4
      Path: /usr/bin/python3
      Jupyter: 4.10.0
      Kernels: python3

[✓] Checking Jupyter engine render....OK

[✓] Checking R installation...........OK
      Version: 4.2.0
      Path: /opt/R/4.2.0/lib/R
      LibPaths:
        - /home/eyayaw/R/x86_64-pc-linux-gnu-library/4.2
        - /opt/R/4.2.0/lib/R/library
      rmarkdown: 2.14

[✓] Checking Knitr engine render......OK
````

**Comment by dragonstyle** [comment, marked as answer]
Please update to the latest version of Quarto which should resolve this.

https://quarto.org/docs/get-started/

**Reply by eyayaw** [reply]
Ok, thanks. However, I have tried to update via the command line. 

```sh
sudo apt upgrade quarto
```
````
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
quarto is already the newest version (0.9.419).
Calculating upgrade... Done
0 to upgrade, 0 to newly install, 0 to remove and 0 not to upgrade.
````

**Reply by dragonstyle** [reply]
Ok not sure where that version comes from- I'd use the link above and try again. Publishing was introduced after the version this is installing.
